src/librustc_codegen_llvm/intrinsic.rs

```diff
@@ -763,6 +763,95 @@ impl IntrinsicCallMethods<'tcx> for Builder<'a, 'll, 'tcx> {
             }
         }
     }
+
+    fn abort(&mut self) {
+        let fnname = self.cx().get_intrinsic(&("llvm.trap"));
```

Comment:
@sunfishcode I think this builder method should be called `trap`, wdyt?

Reply:
In cranelift the `Interrupt` trap can be resumed.

Reply:
Yeah, Cranelift should add an explicit way to request an "abort" trap.

Reply:
I like how bx.abort() lines up with mir::TerminatorKind::Abort. But either way works for me.